package.json

```diff
@@ -73,10 +73,10 @@
     "cordova-android": "~9.0.0",
     "cordova-browser": "~6.0.0",
     "cordova-ios": "~6.1.0",
-    "cordova-osx": "~6.0.0",
-    "cordova-plugin-clipboard": "Jigsaw-Code/outline-cordova-plugin-clipboard#v2.0.0",
+    "cordova-osx": "github:Jigsaw-Code/cordova-osx#ce1b7ad",
```

Comment:
Let's make sure we pull all the latest changes to our fork before submitting this, and update the reference here.